app/controllers/categories_controller.rb

```diff
@@ -1,4 +1,7 @@
-class CategoriesController < ApplicationController
```

Comment:
load and authorize runs load_resource and authorize_resource where it loads appropriate instance variables before running each action. so no need to instantiate variables again and again in actions if you're loading resources as well.